app/js/importExport.js

```diff
@@ -0,0 +1,86 @@
+import FileSaver from 'file-saver'
+
+/**
+ * Import the backup of saved tabs and the accounting information.
+ * If any of the required keys in the backup object is missing, the backup will abort without importing the data.
+ * @param {storageLocal} storageLocal is needed to restore the accounting information
+ * @param {tabManager} tabManager is required to initialize it with the imported saved tabs
+ * @param {Event} event contains the path of the backup file
+ */
+const importData = (storageLocal, tabManager, event) => {
```

Comment:
Can this give the user some feedback about success and failure? After selecting the file, there's no feedback about whether it worked.